recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "1.4.14" %}
+{% set version = "1.5.0" %}
 
 package:
   name: kealib
   version: {{ version }}
 
 source:
   url: https://github.com/ubarsc/kealib/releases/download/kealib-{{ version }}/kealib-{{ version }}.tar.gz
-  sha256: da5d4a540b34afb61665cb7b6bf284825b51464eaf2a23ccca16955e2712cab2
+  sha256: d19a0fb051019f87fe413bda76472bf4fff8fca52ede92e0ffd983caeafd05b8
 
 build:
-  number: 1
+  number: 0
   skip: True  # [win and vc<14]
```

Comment:
If we skip building for `win and vc<14` then `msinttypes` constraints look redundant.

Reply:
yes, I haven't touched this selector.  I will do